.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,48 +3,35 @@ Resolves
 
 ## Submitter Checklist:
 
-- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
+- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue.
 - [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
-- [ ] Added/updated tests for this change (for new code or code which already has tests).
-- Verified that these changes build without errors on
-  - [ ] Android
-  - [ ] iOS
-  - [ ] Linux
-  - [ ] macOS
-  - [ ] Windows
-- Verified that these changes pass automated tests (unit, browser, security tests) on
-  - [ ] iOS
-  - [ ] Linux
-  - [ ] macOS
-  - [ ] Windows
-- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`, `npm run gn_check`)
+- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
+- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`
 - [ ] Ran `git rebase master` (if needed).
-- [ ] Ran `git rebase -i` to squash commits (if needed).
-- [ ] Tagged reviewers and labelled the pull request as needed.
 - [ ] Requested a security/privacy review as needed.
-- [ ] Added appropriate QA labels (`QA/Yes` or `QA/No`) to the associated issue
-- [ ] Added appropriate release note labels (`release-notes/include` or `release-notes/exclude`) to the associated issue
-- [ ] Public documentation has been updated as necessary. For instance:
+- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`) to the associated issue
+
+## Reviewer Checklist:
+
+- [ ] Adequate test coverage exists to prevent regressions
+- [ ] Major classes, functions and non-trivial code blocks are well-commented
+- [ ] Changes in component dependencies are properly reflected in `gn`
+- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
+- [ ] Test plan is specified in PR before merging
+
+## After-merge Checklist:
+
+- [ ] The associated issue milestone is set to the smallest version that the
+  changes has landed on.
+- [ ] All relevant documentation has been updated, for instance:
   - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
   - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
   - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
   - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
   - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
   - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
```

Comment:
not part of this PR, but this should be replaced with https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave

Reply:
Replaced